src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -330,14 +322,13 @@ void AttitudeEstimatorQ::task_main()
 			// Feed validator with recent sensor data
 
 			if (sensors.timestamp > 0) {
-				// Filter gyro signal since it is not fildered in the drivers.
-				_gyro(0) = _lp_gyro_x.apply(sensors.gyro_rad[0]);
-				_gyro(1) = _lp_gyro_y.apply(sensors.gyro_rad[1]);
-				_gyro(2) = _lp_gyro_z.apply(sensors.gyro_rad[2]);
+				_gyro(0) = sensors.gyro_rad[0];
+				_gyro(1) = sensors.gyro_rad[1];
+				_gyro(2) = sensors.gyro_rad[2];
```

Comment:
That will depend on what has been done to the gyro data upstream. If It has been sampled at a high rate, and LPF before downsampling to 250Hz, than additional filtering at 250Hz is unnecessary and also does nothing for aliasing protection.